config/ansible/full_setup.yml

```diff
@@ -0,0 +1,37 @@
+--- # From project directory, run: ansible-playbook config/ansible/full_setup.yml
```

Comment:
this file is a combination of `passenger_test.yml` and `setup.yml`. I didn't delete them in case something doesn't work, its easier to debug separately